src/usr/local/www/services_ntpd.php

```diff
@@ -210,6 +203,13 @@ function build_interface_list() {
 
 $section = new Form_Section('NTP Server Configuration');
 
+$section->addInput(new Form_Select(
+	'enable',
+	'Enable NTP Server',
+	$pconfig['enable'],
+	array('enabled' => 'Enabled', 'off' => 'Disabled')
+));
```

Comment:
I would like to hear from @sbeaver-netgate about this change. Several screens use a checkbox to enable/disable services